README.md

```diff
@@ -59,9 +59,19 @@ wikibaseintegrator~=0.11.3
 # WikibaseIntegrator / WikidataIntegrator #
 
 WikibaseIntegrator (wbi) is a fork of [WikidataIntegrator](https://github.com/SuLab/WikidataIntegrator) (wdi) whose
-purpose is to focus on compatibility with Wikibase. There have been many improvements which have led to radical changes
-in the code. See the [release notes](https://github.com/LeMyst/WikibaseIntegrator/releases) to find out what has
-changed.
+purpose is to focus on an improved compatibility with Wikibase and adding missing functionalities.
+The main differences between these two libraries are :
+
+* A complete rewrite of the library with an object-oriented architecture allowing for easy interaction, data validation
+  and extended functionality
```

Comment:
WikdataIntegrator is also object-oriented (e.g. the `WDProperty` class inherits from `WDBaseDataType`), how is WikibaseIntegrator more object-oriented? How does WBI allow for easy interaction? What is meant by data validation? I don't think this library implements property constraints or entity schemas? What is meant by extended functionality? Can I implement my own entity type without having to patch WBI?